src/test/java/itest/TargetRecordingOptionsIT.java

```diff
@@ -63,6 +64,35 @@
             String.format("/api/v2/targets/%s/recordingOptionsList", SELF_REFERENCE_TARGET_ID);
     static final String RECORDING_NAME = "test_recording";
 
+    @AfterAll
+    static void resetDefaultRecordingOptions() throws Exception {
+        CompletableFuture<JsonObject> getResponse = new CompletableFuture<>();
+        MultiMap form = MultiMap.caseInsensitiveMultiMap();
+        form.add("maxAge", "0");
+        form.add("toDisk", "false");
+        form.add("maxSize", "0");
+
+        webClient
+                .patch(OPTIONS_REQ_URL)
+                .sendForm(
+                        form,
+                        ar -> {
+                            if (assertRequestStatus(ar, getResponse)) {
+                                MatcherAssert.assertThat(
+                                        ar.result().statusCode(), Matchers.equalTo(200));
+                                MatcherAssert.assertThat(
+                                        ar.result().getHeader(HttpHeaders.CONTENT_TYPE.toString()),
+                                        Matchers.equalTo(HttpMimeType.JSON.mime()));
+                                getResponse.complete(ar.result().bodyAsJsonObject());
+                            }
+                        });
+
+        JsonObject expectedGetResponse =
+                new JsonObject(Map.of("maxAge", 0, "toDisk", false, "maxSize", 0));
+
+        MatcherAssert.assertThat(getResponse.get(), Matchers.equalTo(expectedGetResponse));
```

Comment:
I think we can skip the test assertion in this `AfterAll`, and the `BeforeAll` too if you add it.

Reply:
Sounds good